test/system/005-info.bats

```diff
@@ -328,4 +328,48 @@ EOF
     CONTAINERS_STORAGE_CONF=$PODMAN_TMPDIR/storage.conf run_podman $safe_opts info
 }
 
+@test "podman info --rewrite-config respects new runRoot" {
+    skip_if_remote "Test uses nonstandard paths for c/storage directories"
+
+    # Create temporary storage directories
+    GRAPHROOT=$(mktemp -d)
+    RUNROOT_A=$(mktemp -d)
+    RUNROOT_B=$(mktemp -d)
+
+    STORAGE_CONF1=$PODMAN_TMPDIR/storage1.conf
+    STORAGE_CONF2=$PODMAN_TMPDIR/storage2.conf
+
+    cat >$STORAGE_CONF1 <<EOF
+[storage]
+driver="$(podman_storage_driver)"
+graphroot = "$GRAPHROOT"
+runroot = "$RUNROOT_A"
+[storage.options]
+EOF
+
+    cat >$STORAGE_CONF2 <<EOF
+[storage]
+driver="$(podman_storage_driver)"
+graphroot = "$GRAPHROOT"
+runroot = "$RUNROOT_B"
+[storage.options]
+EOF
+
+    # First, verify original runRoot is used with first config
+    run_podman --storage-conf=$STORAGE_CONF1 info
+    echo "$output" | grep -q "RunRoot: $RUNROOT_A"
+
+    # Second config should be ignored; still original runRoot
+    run_podman --storage-conf=$STORAGE_CONF2 info
+    echo "$output" | grep -q "RunRoot: $RUNROOT_A"
+
+    # With rewrite-config flag, new runRoot should be used
+    run_podman --storage-conf=$STORAGE_CONF2 --rewrite-config info
+    echo "$output" | grep -q "RunRoot: $RUNROOT_B"
+
+    rm -rf "$RUNROOT_A"
+    rm -rf "$RUNROOT_B"
+    rm -rf "$GRAPHROOT"
+}
+
```

Comment:
Can you do tests for the error conditions as defined in this [if](https://github.com/containers/podman/pull/27157/files#diff-38ec5b000c4219e9e9979896785dd80f920de8ce3c56255ec30adc79b5fdd209R405)?